src/cli.ts

```diff
@@ -1,164 +1,82 @@
-      'The prefix for the MQTT unique to this tenant for sending and receiving device messages',
-      env.MQTT_MESSAGES_PREFIX,
+    .addOption(
+      new Option(
+        '-c, --certificate-type <certificateType>',
+        'Specifies the type of certificate you want created for your new device: self-signed certificate or one for Just-In-Time-Provisioning (JITP). The latter is discouraged, and is mainly for internal Nordic Semiconductor use.',
+      ).choices(['Self-Signed', 'JITP']).default('Self-Signed'),
     )
     .option(
-      '-s, --services <services>',
-      'Comma-delimited list of services to enable. Any of: [gps,gnss,acc,temp,device,rsrp,location,log,alert]',
+      '-p, --prevent-association',
+      'Specifies that when your device with a Just-In-Time-Provisioning (JITP) certificate connects to the MQTT broker (before it is ever associated with your team), it should only connect, i.e., not get associated. This option is only applicable when the "-c jitp" option is specified and the device has not already been added to (associated with) your team. This is mainly for internal Nordic Semiconductor use to test JITP issues.',
+      false,
+    )
+    .addOption(
+      new Option(
+        '-s, --sensors <sensors>',
+        `A comma-separated list of sensors you want your device to simulate. Valid choices: ${
+          sensorOptions.join(', ')
+        }. See https://github.com/nRFCloud/application-protocols for more information about sensor messages.`,
+      )
+        .argParser((value) => {
+          const values = value.split(',');
+          values.forEach((val) => {
+            if (!sensorOptions.includes(val)) {
+              throw new Error(`Invalid choice: ${val}. Allowed choices are: ${sensorOptions.join(', ')}`);
+            }
+          });
+          return values;
+        })
+        .default([]),
     )
     .option(
       '-f, --app-fw-version <appFwVersion>',
       'Version of the app firmware',
       '1',
     )
-    .option(
-      '-h, --api-host <apiHost>',
-      'API host for nRF Cloud',
-      env.API_HOST ? env.API_HOST : 'https://api.nrfcloud.com',
-    )
-    .option(
-      '-a, --onboard <onboardingType>',
-      'Onboard the device with "jitp", or "preconnect"',
-      handleOnboardingType,
-    )
-    .option('-v, --verbose', 'output debug info', false)
-    .option(
-      '-t, --app-type <appType>',
-      'Specifies the shadow to use. For custom shadow, pass a JSON-encoded shadow object or relative path to json file. Otherwise, pass "mss" or "atv2" to automatically generate a conformal shadow',
-      handleAppType,
-    )
-    .option(
-      '-p, --job-execution-path <jobExecutionPath>',
-      'Specifies an unhappy job execution path for a fota update. View the "Use an unhappy path for FOTA execution" section of the README for more details.',
-      handleJobExecution,
-    )
+    .addOption(new Option(
+      '-a, --app-type <appType>',
```

Comment:
The old `-t` option allowed for a user to pass a custom shadow, instead of simulating an `mss` or `atv2` shadow. So this PR removes that feature? Not sure if it was necessary anyway, it was a new feature recently. Users could create a simulated device and later change the shadow by other means.

Reply:
Yeah, that was my thinking. Or they can just modify the code themselves.